File: CEncryption.java
import java.util.Scanner;

// A quick and messy method of dealing with a ceasar encryption.

public class CEncryption {

	static String[] letters = {"A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"};
	
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.println("What is the phrase you want to change?");
		String original = scan.nextLine();
		System.out.println("How many rotations?");
		int rotations = scan.nextInt();
		char[] originalChange = original.toCharArray();
		original = original.toUpperCase();
		for(int i = 0; i < original.length(); i++) {
			for(int j = 0; j < letters.length; j++) {
				if(original.charAt(i) == letters[j].toCharArray()[0]) {
					originalChange[i] = letters[(j + rotations) % 26].toCharArray()[0];
				}
			}
		}
		System.out.println(originalChange);
	}

}

File: CEncryption.py
# A quick and messy method of dealing with a ceasar encryption.
letters = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"]
original = input("What is the sentence? ")
original = original.upper()
originalList = list(original)
rotations = int(input("How many rotations? "))
for i in range(0,len(original)):
    for j in range(0,len(letters)):
        if originalList[i] == letters[j]:
            originalList[i] = letters[(j+rotations)%26]
            break
for i in range(0,len(originalList)):
    print(originalList[i], end="")
